doc/source/user_guide/changelog/v1.2.0.rst

```diff
@@ -0,0 +1,42 @@
+What's new in 1.2.0 (14 August 2024)
+-----------------------------------
+
+These are the changes in icepyx 1.2.0 See :ref:`release` for a full changelog
+including other versions of icepyx.
+
+
+New Features
+~~~~~~~~~~~~
+
+- Replace `setup.py` with equivalent `pyproject.toml` (#539)
```

Comment:
We could consider this "maintenance"! No strong feelings.

Reply:
Agreed - there weren't any other new features though! : )